Enterprise-CMCS/eAPD
# Fixed issue with adding and removing years breaking Key State Personnel

Resolves #4572 

### Description
When the updates were made the the key state personnel model, the methods that update the APD when an FFY is added or removed weren't updated, so it would break the Key State Personnel page when you changed the years and tried to go back. I just added the new fields to the add and remove methods.

### Automated test cases written

| Given | When | Then | Type (jest, tap, cypress) |
| ----- | ---- | ---- | ------------------------- |
| a HITECH APD | a year is added | it updates the key state personnel correctly | jest |
| a HITECH APD | a year is removed | it updates the key state personnel correctly | jest |
| a MMIS APD | a year is added | it updates the key state personnel correctly | jest |
| a MMIS APD | a year is removed | it updates the key state personnel correctly | jest |

### Steps to manually verify this change

1. Create an MMIS APD
2. Add a Key State Personnel
3. Add an FFY
4. Go back to Key State Personnel and make sure it updated correctly
5. Remove an FFY (the first one)
6. Go back to Key State Personnel and make sure it updated correctly

### This pull request is ready to code review when

      tests, refactor

### This pull request is ready to test when


### This pull request is ready to review when the QA has


### This pull request can be merged when
